source/server/http/admin.cc

```diff
@@ -47,6 +49,59 @@ using namespace rapidjson;
 namespace Envoy {
 namespace Server {
 
+namespace {
+
+const char kEnvoyFavicon[] =
```

Comment:
nit: We don't do "k" prefix for constance in the codebase. Either PascalCase or all caps is fine (We are not super consistent on this). Same below. 

Additionally comment on this one in particular: Can you add comments on how this was generated (commands, what it is, etc. if this needs to be recreated in the future)?

Reply:
done....I'm slightly troubled by all-caps because I kind of feel like those are the domain of preprocessor macros, which are hard to override without other preprocessor macros, which don't respect namespaces.

PascalCase is also the domain of types in the style guide, but I went with that as I feel clashes would be easier to diagnose than clashes with C macros.

I have also seen snake_case for global constants in my travels through this codebase, in source/common/common/version_linkstamp.

I added the story of the favicon :)